### ISSUE TITLE
python3.7=3.7.2-1+xenial1, xenial: ModuleNotFoundError: cmath

### ISSUE THREAD

**Issue by webknjaz** [deadsnakes/issues#78]
Refs:
* https://twitter.com/webKnjaZ/status/1083804169776103426
* https://stackoverflow.com/a/51716364/595220
* #68

P.S. Microfeedback: it's counterintuitive where to report issues. Took some time to figure out :)

**Comment by asottile** [comment]
not sure how to improve on the micro feedback :S

- The [ppa page](https://launchpad.net/~deadsnakes/+archive/ubuntu/ppa) includes a "Reporting Issues" which links here
- https://github.com/deadsnakes has the issues repo pinned here
- and the [issue template](https://github.com/deadsnakes/issues/blob/master/.github/ISSUE_TEMPLATE.md) guides through the creation process

working on the fix for `cmath` now -- it looks like just a revert of the least fix :laughing:

**Comment by asottile** [comment]
found the fix, the name of the make variable changed again -- building and verifying the patch right now and then I'll upload the fixed release!

**Comment by asottile** [comment]
should be fixed in `3.7.2-1+xenial2` -- thanks again for the issue!  It should be available from launchpad within ~45 minutes

```
root@00e2b8aedce7:/# python3.7
Python 3.7.2 (heads/ubuntu/xenial-dirty:b3af486, Jan 11 2019, 21:31:15) 
[GCC 5.4.0 20160609] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import cmath
>>> 
```

**Comment by webknjaz** [comment]
Oh thank you very much!
